root/release/edit/information.tt

```diff
@@ -314,110 +314,253 @@
+      <!-- /ko -->
+      <!-- ko ifnot: $data.gid -->
+        <p>
+          [% l('The {doc|release group} groups releases (like different editions of a single album) into a single logical entity.',
+               { doc => { href => doc_link('Release_Group'), target => '_blank' } }) %]
+        </p>
+        <p>
+          [% l('Please see the {doc|style guidelines} for more information.',
+               { doc => { href => doc_link('Style/Release_Group'), target => '_blank' } }) %]
+        </p>
+      <!-- /ko -->
     <!-- /ko -->
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.dateBubble">
-    <p data-bind="visible: target() && target().hasAmazonDate()">
-      [% l('Warning! "1990-10-25" is the bogus date that Amazon gives to all releases for which they don\'t know the actual date. <strong>Please use this date only if you\'re certain this date is correct!</strong>') %]
+  <div class="bubble" data-bind="bubble: $root.primaryTypeBubble">
+    <p>
+      [% l('The {doc|primary type} describes how the release group is categorized.',
+           { doc => { href => doc_link('Release_Group/Type'), target => '_blank' } }) %]
     </p>
-    <p data-bind="visible: target() && target().hasJanuaryFirstDate()">
-      [% l('Note! If you do not know the month or day of release, please leave them empty.  January 1st is not often the actual release date, please double check that you have entered the release date correctly.') %]
+  </div>
+
+  <div class="bubble" data-bind="bubble: $root.secondaryTypesBubble">
+    <p>
+      [% l('The {doc|secondary types} are additional attributes describing the release group.',
+           { doc => { href => doc_link('Release_Group/Type'), target => '_blank' } }) %]
     </p>
   </div>
 
-  <div class="bubble" data-bind="bubble: $root.labelBubble">
-    <!-- ko with: target() && target().label() -->
-      <!-- ko if: $data.gid -->
-        <p data-bind="html: $data.selectionMessage()"></p>
-      <!-- /ko -->
+  <div class="bubble" data-bind="bubble: $root.statusBubble">
+    <!-- ko if: target() && target().statusID() == 4 -->
+      <p>
+        [% l('This release status should be used for unofficial translations
+              and transliterations of tracklists and release titles, and does
+              not denote a separate real release. It should not be used to
+              denote bootlegs, mixtape/street albums, demos, or digital
+              albums. Be sure to link to the corresponding real release with
+              the {url|transl(iter)ation relationship}.',
+             { url => c.uri_for('/relationship/fc399d47-23a7-4c28-bfcf-0607a562b644') }) %]
+      </p>
     <!-- /ko -->
-    <!-- ko if: target() && catNoLooksLikeASIN(target().catalogNumber()) -->
+    <!-- ko ifnot: target() && target().statusID() == 4 -->
       <p>
-        [% l('The catalog number you have entered looks like an Amazon ASIN.') %]
+        [% l('The {doc|status} describes how “official” the release is.',
```

Comment:
Ideally this would also be displaying descriptions loaded from the DB (every type has one), but at least I'd suggest always displaying the two lines you're adding here, and then *additionally* displaying the pseudo-release text for pseudo-releases.